app/web/features/search/MapWrapper.tsx

```diff
@@ -105,11 +105,7 @@ export default function MapWrapper({
    * User clicks on a user on map
    */
   const handleMapUserClick = useCallback(
-    (
-      ev: MapMouseEvent & {
-        features?: Feature<Geometry, GeoJsonProperties>[] | undefined;
-      } & MaplibreEvent
-    ) => {
+    (ev: MapLayerMouseEvent) => {
```

Comment:
See `MapLayerMouseEvent` is the same type here: https://github.com/maplibre/maplibre-gl-js/blob/91e49458f9c06726550510e94ee549c380c81f38/src/ui/events.ts#L17